fdbcli/MetaclusterCommands.actor.cpp

```diff
@@ -287,9 +287,47 @@ ACTOR Future<bool> metaclusterStatusCommand(Reference<IDatabase> db, std::vector
 
 	state bool useJson = tokens.size() == 3;
 
+	state Reference<ITransaction> tr = db->createTransaction();
+	tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
+	tr->setOption(FDBTransactionOptions::RAW_ACCESS);
+	state Future<Optional<MetaclusterRegistrationEntry>> metaclusterRegistrationFuture =
+	    MetaclusterAPI::getMetaclusterRegistrationEntryTransaction(tr);
```

Comment:
Similar to the comment about `listClustersTransaction` and `listClustersWithTxn()`. This `MetaclusterAPI::getMetaclusterRegistrationEntryTransaction` handles retry, while the other new helper function does not. If retrying is not intended or necessary, this can be simplified according to your review comments.